architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator. The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available as part of Red Hat Advanced Cluster Management for Kubernetes, or as a standalone operator that is supported as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription.
+
+If you use Red Hat Advanced Cluster Management, multicluster engine is included and provides the following capabilities:
+
```

Comment:
I would make 2 sentences:
"If you use Red Hat Advanced Cluster Management, the multicluster engine is already included as a prerequisite. 
The multicluster engine provides the following capabilities:"

Reply:
Can we add a hyperlink to RHACM for the RHACM mention?